src/Namotion.Reflection/XmlDocsFormatting.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Namotion.Reflection
+{
+    /// <summary>
+    /// Contains the logic to maintain formatting of XML-doc elements.
+    /// </summary>
+    internal static class XmlDocsFormatting
+    {
+        private static readonly IDictionary<XmlDocsFormattingMode, Func<StringBuilder, XElement, StringBuilder>> formattingFunctions =
```

Comment:
should use concrete type so JIT doesn't need to worry about virtual dispatch

Reply:
Changed to `Dictionary<...>`